spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/SqsContainerOptionsBuilder.java

```diff
@@ -65,4 +66,11 @@ SqsContainerOptionsBuilder messageSystemAttributeNames(
 	 */
 	SqsContainerOptionsBuilder queueNotFoundStrategy(QueueNotFoundStrategy queueNotFoundStrategy);
 
+	/**
+	 * Set the {@link AcknowledgementMode} for the container.
+	 * @param acknowledgementMode the acknowledgement mode.
+	 * @return this instance.
+	 */
+	SqsContainerOptionsBuilder acknowledgementMode(AcknowledgementMode acknowledgementMode);
```

Comment:
Not sure why we need this - we already have `acknowledgeMode` in the `AbstractContainerOptions` class.

Reply:
you're right. thanks!